src/configure/configure.ts

```diff
@@ -254,7 +255,7 @@ class Orchestrator {
     }
 
     private async analyzeNode(node: any): Promise<GenericResource> {
-        if (!!node && !!node.fullId) {
+        if (!!node && !!node.cloudResource.id) {
```

Comment:
Does AKS extension publish any type definition for cluster node type? If so, may be we should use that.

Reply:
They have defined their own namespace which contains the structure of CloudExplorerResourceNode which we are getting in our extension. The cloudResource is of type any.

![image](https://user-images.githubusercontent.com/58769601/94138963-8776bd00-fe86-11ea-933c-54858585124c.png)